src/main/scala/com/fulcrumgenomics/fastq/DemuxFastqs.scala

```diff
@@ -331,15 +337,17 @@ class DemuxFastqs
      """)
  val includeAllBasesInFastqs: Boolean = false,
  @deprecated(message="Use outputStandards instead", since="1.3.0")
- @arg(doc="Output FASTQs according to Illumina BaseSpace Sequence Hub naming standards.  This is differfent than Illumina naming standards.",
+ @arg(doc="Output FASTQs according to Illumina BaseSpace Sequence Hub naming standards.  This is different than Illumina naming standards.",
    mutex=Array("fastqSkipReadNumbers", "fastqIncludeSampleBarcodes", "illuminaFileNames"))
  val illuminaStandards: Boolean = false,
  @arg(doc="Include a trailing /1 or /2 for R1 and R2 in the FASTQ read name.", mutex=Array("illuminaStandards"))
  var fastqSkipReadNumbers: Boolean = false,
  @arg(doc="Update the sample barcode in the FASTQ header.", mutex=Array("illuminaStandards"))
  var fastqIncludeSampleBarcodes: Boolean = false,
  @arg(doc="Name the output files according to the Illumina file name standards.", mutex=Array("illuminaStandards"))
- var illuminaFileNames: Boolean = false
+ var illuminaFileNames: Boolean = false,
+ @arg(doc="Keep only passing filter reads if true, otherwise keep all reads. Passing filter reads are determined from the comment in the FASTQ header")
```

Comment:
```suggestion
 @arg(doc="Keep only passing filter reads if true, otherwise keep all reads. Passing filter reads are determined from the comment in the FASTQ header.")
```